src/ResultPage.js

```diff
@@ -8,17 +8,19 @@ import data from './data/measures.json'
 import { VirusWhiteIcon } from './lib/Icons'
 import scrollUp from './lib/scrollUp'
 
-function ResultPage({ setUserPlace, countyData, status }) {
+function ResultPage({ startSearch, countyData, status }) {
     const [countyClassification, setCountyClassification] = useState('')
     const [measures, setMeasures] = useState([])
 
     const history = useHistory()
 
     let { id } = useParams()
 
+    // eslint-disable-next-line react-hooks/exhaustive-deps
     useEffect(() => classifyCountyIncidence(), [countyData])
     useEffect(() => {
-        status !== 'loaded' && setUserPlace(id)
+        status !== 'loaded' && startSearch(id)
+        // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [id])
 
     useEffect(() => setMeasures(data), [])
```

Comment:
You can think about rearranging this line and put it above the other ``useEffects``. since this ``useEffect`` has an empty dependency array and thus is more "general"